plugins/scaffolder-node/api-report.md

```diff
@@ -315,8 +315,18 @@ export interface TaskBroker {
   // (undocumented)
   get(taskId: string): Promise<SerializedTask>;
   // (undocumented)
-  list?(options?: { createdBy?: string; status?: TaskStatus }): Promise<{
+  list?(options?: {
+    createdBy?: string | string[];
+    status?: TaskStatus | TaskStatus[];
+    limit?: number;
+    offset?: number;
+    order?: {
+      order: 'asc' | 'desc';
+      field: string;
+    }[];
+  }): Promise<{
     tasks: SerializedTask[];
+    totalTasks: number;
```

Comment:
This is also a breaking change given that anyone already implementing list would have to start supplying this field.